static/data/index.mjs

```diff
@@ -0,0 +1,91 @@
+export const imageDimensions = {
+  '0level': 0,
```

Comment:
the variable name should be `dimensionsByLevel` and keys 0,1,2,3. A bit less clunky

Reply:
100% agree, done.